### PR TITLE
tutorial improvements

### PR DESCRIPTION
**Pull request by Anton-4**
Some follow-up on #6952.

### CODE REVIEW

File: www/content/tutorial.md [hunk 1]
```diff
@@ -820,18 +820,29 @@ Result.isOk (List.get ["a", "b", "c"] 1)
 ```
 
 ```roc
-Result.try (Str.toU64 "2") \idx -> List.get ["a", "b", "c", "d"] idx
```
Comment (Anton-4) [inline www/content/tutorial.md:823]: I expect this will be hard to parse for beginners, I tried a formatting change, but I think the separate `listGet` function is best in this case.

File: www/content/tutorial.md [hunk 2]
```diff
@@ -1596,7 +1607,7 @@ main =
     Stdout.line! "Hello, World!"
 ```
 
-The `Stdout.line` function takes a `Str` and writes it to [standard output](<https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stdout)>). (We'll discuss the `!` part later.) `Stdout.line` has this type:
+The `Stdout.line` function takes a `Str` and writes it to [standard output](<https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stdout)>), we'll [discuss the `!` part later](https://www.roc-lang.org/tutorial#the-!-suffix). `Stdout.line` has this type:
```
Comment (Anton-4) [inline www/content/tutorial.md:1610]: Just adding a link here

File: www/content/tutorial.md [hunk 3]
```diff
@@ -820,18 +820,29 @@ Result.isOk (List.get ["a", "b", "c"] 1)
 ```
 
 ```roc
-Result.try (Str.toU64 "2") \idx -> List.get ["a", "b", "c", "d"] idx
-# returns `Ok "c"`
+Result.try (Str.toU64 "2") listGet
+
+listGet = U64 -> Result Str [OutOfBounds]
```
Comment (ageron) [inline www/content/tutorial.md:825]: If think there's a typo here: it should be `listGet :` instead of `listGet =`

Comment (lukewilliamboswell) [reply www/content/tutorial.md:825]: @Anton-4 this type is still here

Comment (Anton-4) [reply www/content/tutorial.md]: Oops, thanks for noticing, I forgot to test the code.

**Review by lukewilliamboswell — APPROVED** [review]
